packages/calcite-components/src/components/combobox/combobox.scss

```diff
@@ -165,7 +165,7 @@
 }
 
 .label {
-  @apply pointer-events-none max-w-full flex-auto truncate p-0 font-normal;
```

Comment:
Do we know why pointer events were disabled here?

Reply:
I'm not sure, and I don't see it doing anything. We are able to select by clicking on label on dev, and that should be the behavior?

Reply:
This change doesn't affect `multiple` selection-mode, which is the subject of this PR, so my recommendation is to revert this change.  I tested it both with and without this change and it doesn't seem to affect it either way, but because the `span.label` this style applies to only renders in single selection-mode, its a good idea to leave this style as-is.